bin/start.js

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env node
 
+var log = require('npmlog')
+log.level = 'warn'
+
 var cli = require('../cli')
 
 var emoji = require('node-emoji')
 
 cli(function (error, server) {
   if (error) {
+    log.warn('DB url config', 'If you are using --dbUrl, --dbUrlUsername, --dbUrlPassword, you should provide your authentication details without URL encoding')
```

Comment:
Agree, but it’s the wrong place to show this warning. It should go to the place the parsing is happening